### ISSUE TITLE
Block $schema at io-package.json

### ISSUE THREAD

**Issue by mcm1957** [ioBroker/ioBroker.repochecker#328]
$schema gehört NICHT in io-package.json

https://forum.iobroker.net/topic/77124/fehlermeldungen-dev-server-beim-%C3%B6ffen-der-config/17

**Comment by oweitman** [comment]
$schema ist ein gültiger begriff für eine json Datei 
https://json-schema.org/learn/getting-started-step-by-step#create-a-schema-definition
Sie verweist auf die Schema Definition gegen die die aktuelle json Datei geprüft werden kann. Wenn da ein Fehler ist, dann liegt es an der Prüfung.

**Comment by mcm1957** [comment]
Ja, gültig ist sie. Nur hat das nix im jsonConfig.json zu suchen.
Die Prüfdatei sollte NICHT in jsonConfig.json "umgebogen" werden.

**Comment by mcm1957** [comment]
added 3.3.0
